packages/documentation/copy/en/handbook-v2/Basics.md

```diff
@@ -398,7 +398,7 @@ So running `tsc --target es2015 hello.ts` gives us the following output:
 
 ```js
 function greet(person, date) {
-  console.log(`Hello ${person}, today is ${date.toDateString()}!`);
+    console.log(`Hello ${person}, today is ${date.toDateString()}!`);
```

Comment:
```suggestion
  console.log(`Hello ${person}, today is ${date.toDateString()}!`);
```

Seems like other examples are using two spaces in this file, so best to be consistent.